/border_checker:
  ros__parameters:
    filter_field: z
    filter_limits_max: 20.0
    filter_limits_min: -0.2
    filter_type: passthrough
    mean_k: 50
    output_topic: rest
    standard_dev_thresh: 1.0
    use_sim_time: false
